crawler/__init__.py

```diff
@@ -26,7 +27,8 @@ def fetch(cls, sources, options):
         ret = list()
         for source in sources:
             resp = requests.get(source, headers=config.req_header)
-            if options["mode"] == "xml": ret.append(BeautifulSoup(resp.text))
+            if options["mode"] == "xml": ret.append(BeautifulSoup(resp.text.replace("<![CDATA[", "").replace("]]>", ""), "xml"))
```

Comment:
Line break here to fit within 80 chars. (I think github's style guide is 80 for Python?)